cli/.js/src/main/scala/aqua/builder/ConsoleServiceBuilder.scala

```diff
@@ -0,0 +1,35 @@
+package aqua.builder
+
+import aqua.io.OutputPrinter
+import aqua.js.{CallJsFunction, CallServiceHandler, FluencePeer}
+import aqua.model.func.Call
+import aqua.model.func.raw.CallServiceTag
+import aqua.model.{LiteralModel, VarModel}
+
+import scala.scalajs.js.JSON
+
+// Service to print any variables
+class ConsoleServiceBuilder(serviceId: String, fnName: String) {
```

Comment:
The comment does not match the class name and what the class does does not match it either.